.github/workflows/wheels-dependencies.sh

```diff
@@ -51,6 +51,7 @@ LIBWEBP_VERSION=1.5.0
 BZIP2_VERSION=1.0.8
 LIBXCB_VERSION=1.17.0
 BROTLI_VERSION=1.1.0
+LIBAVIF_VERSION=1.2.1
```

Comment:
```suggestion
LIBAVIF_VERSION=1.3.0
```
libavif 1.3.0 has been released - https://github.com/AOMediaCodec/libavif/releases/tag/v1.3.0

Reply:
Thanks for the suggested change. I suggest two related changes if you will only be building with libavif v1.3.0 or later.

1. Please delete all occurrences of `CONFIG_AV1_DECODER=0` in your build files (including the comments). libavif v1.3.0 or later sets `CONFIG_AV1_DECODER` to 0 if you pass `-DAVIF_CODEC_AOM_DECODE=OFF` to libavif.
2. Please try removing `"-DCMAKE_POLICY_VERSION_MINIMUM=3.5",`. We fixed all the old `cmake_minimum_required()` versions in libavif v1.3.0. You should not need this workaround now.

Reply:
From what I see, these suggestions have been applied.